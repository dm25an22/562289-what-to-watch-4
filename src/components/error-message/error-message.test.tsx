import * as React from 'react';
import * as renderer from "react-test-renderer";
import ErrorMessage from './error-message';

it(`render ErrorMessage component`, () => {
  const tree = renderer.create(
      <ErrorMessage errorCode={404} />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
