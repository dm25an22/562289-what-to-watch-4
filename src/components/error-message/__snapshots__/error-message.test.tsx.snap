// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render ErrorMessage component 1`] = `
Array [
  <h1
    style={
      Object {
        "fontSize": "60px",
      }
    }
  >
    Error loading data from server: 
    404
     
  </h1>,
  <p
    style={
      Object {
        "fontSize": "40px",
      }
    }
  >
    Try reloading the page
  </p>,
]
`;
